File: 8-kyu/return-the-day-/python/solution.py
def whatday(num):
    day_dict = {1: 'Sunday', 2: 'Monday', 3: 'Tuesday', 4: 'Wednesday', \
                5: 'Thursday', 6: 'Friday', 7: 'Saturday'}
    return day_dict.get(num, 'Wrong, please enter a number between 1 and 7')

File: 8-kyu/return-the-day-/javascript/solution.js
def whatday(num):
    day_dict = { 1: 'Sunday', 2: 'Monday', 3: 'Tuesday', 4: 'Wednesday', 5: 'Thursday', 6: 'Friday', 7: 'Saturday' }
    return 'Wrong, please enter a number between 1 and 7' if num not in day_dict else day_dict[num]
    